examples/mobile/HomeAppServer/bin/www

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env node
+
+/**
+ * Module dependencies.
+ */
+
+var app = require('../app');
+var debug = require('debug')('homeappserver:server');
+var http = require('http');
+
+/**
+ * Get port from environment and store in Express.
+ */
+
+var port = normalizePort(process.env.PORT || '3000');
```

Comment:
Is the 3000 port fixed?

Reply:
it is default port used by express server
You can change customize port when starting server using PORT env variable e.g.
`PORT=8000 npm start`